textile/features.textile

```diff
@@ -526,16 +527,16 @@ h3(#realtime-connection). Connection
 ** @(RTN20c)@ When @CONNECTING@, if the operating system indicates that the underlying internet connection is now available, the client should restart the pending connection attempt
 * @(RTN16)@ @Connection@ recovery:
 ** @(RTN16i)@ Connection recovery is similar to connection resumption (see "RTN15c":#RTN15c), except that instead of the library resuming from a time at which that library instance was previously connected, it is doing so from external state provided in the client options, "(TO3i)":#TO3i. Since the library has no state at the time of connection, the channels must be explicitly attached by the user; continuity preservation is achieved by the @channelSerial@s for each channel being stored in the recovery key.
```

Comment:
Small doubt - 
Not sure why user need to manually attach channels in recover. Seems they need to iterate over each channel and attach them explicitly once recover is successful. 
In case of resume, it will automatically attach the channels.

Reply:
think about it. In the case of recover the subscribe listeners don't exist yet, the user needs to re-subscribe. If we automatically attached the channels, the sdk might receive and process all the resume message backlog before anyone's attached any subscribe listeners yet, so the messages would be discarded

Reply:
fair 👍